lib/response_handler.rb

```diff
@@ -1,38 +1,58 @@
+require 'response'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.